### DISCUSSION TITLE
Is there an estimated time for 4.0 official release？

### DISCUSSION THREAD

**Discussion by BrunuhVille** [transmission/transmission#4342] [Q&A]
I am too excited for the 4.0 official release.
Is there an estimated time like within 3 months or within 6 months?
Thanks

**Comment by Coeur** [comment]
When this list will be empty:
https://github.com/transmission/transmission/milestones/4.0.0

So it mostly depends on @ckerr ;)

**Comment by ckerr** [comment]
I don't have an estimated time.
